src/main/java/de/tum/in/www1/artemis/service/ModelingSubmissionService.java

```diff
@@ -207,6 +210,14 @@ public ModelingSubmission save(ModelingSubmission modelingSubmission, ModelingEx
         modelingSubmission.setParticipation(participation);
         modelingSubmission = modelingSubmissionRepository.save(modelingSubmission);
 
+        // versioning of submission
+        try {
+            submissionVersionService.save(modelingSubmission, username);
```

Comment:
please only invoke this for team exercises (pass TeamExercise in the method)

Reply:
Done